### DISCUSSION TITLE
Anyone else get a torrent of bogus notifications yesterday?

### DISCUSSION THREAD

**Discussion by SpenceKonde** [SpenceKonde/ATTinyCore#566] [General]
I got a ton of email notifications about an individual (always the same dude) making 1-word replies, usually "nice" in dozens of issues in my cores, mostly this one. Did others who were "following" threads here get this? Am I missing some obvious way to see this discussion in "view on github"?

**Comment by sleemanj** [comment]
Yes I did.  I assume it's a spammer that got nuked.

**Reply by prandeamus** [reply]
I saw it happen yesterday (mid-day and later UK time) and reported the spammer to github management. I can't remember exactly how - was there a link on the home page or something. Maybe they removed the comments overnight?
